p/p_test.go

```diff
@@ -8,5 +8,5 @@ import (
 
 func TestIt(t *testing.T) {
 	p.DoIt()
-	p.ReDoIt()
+	p.DoItAgain()
```

Comment:
;;; Type C-c C-c to include your comment locally
;;; Type C-c C-k to abort

```suggestion
	p.DoItAgain()
```

Maybe it's OK?